tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/OrganisationControllerTests.cs

```diff
@@ -209,5 +209,65 @@ public async Task GetIdByAsync_VerifyMethodIsCalledOnce_VerifiesMethod()
 
             mockGetOrganisation.Verify(x => x.GetByIdAsync(expectedId), Times.Once);
         }
+
+        [Test]
+        public async Task UpdateOrganisationByIdAsync_UpdateOrganisation_ReturnsStatusNoContent()
+        {
+            using var controller = OrganisationControllerBuilder.Create().WithUpdateOrganisation(new Organisation()).Build();
+
+            var response = await controller.UpdateOrganisationByIdAsync(Guid.Empty, new UpdateOrganisationViewModel());
+
+            response.Should().BeOfType<NoContentResult>();
+        }
+
+        [Test]
+        public async Task UpdateOrganisationByIdAsync_UpdateOrganisation_ReturnsStatusNotFound()
```

Comment:
Maybe add a bit more in the method name, as at the moment it looks like you are doing the same thing (as the method above) but returning different things

Reply:
👍